src/context.rs

```diff
@@ -192,15 +195,41 @@ pub struct Context<'a, 'b: 'a, 's> {
 
 ///A reader for a request body.
 pub struct BodyReader<'a, 'b: 'a> {
-    request: HttpReader<&'a mut BufReader<&'b mut NetworkStream>>
+    reader: HttpReader<&'a mut BufReader<&'b mut NetworkStream>>,
+    is_multipart: bool,
```

Comment:
Alright, it may be a good idea to change it, then. The way your API is designed made me think that it could still work without the boundary, or will this somehow be supported in the future?